kayenta-core/src/main/java/com/netflix/kayenta/metrics/MetricsRetryConfigurationProperties.java

```diff
@@ -0,0 +1,28 @@
+package com.netflix.kayenta.metrics;
```

Comment:
Please add a copyright header to this file.

Reply:
added

Reply:
Thanks. Please use 2019 instead of 2017 and your name or company name instead of Google.